src/packages/xroad/debian/changelog

```diff
@@ -1,3 +1,16 @@
+xroad (6.14.0) trusty; urgency=low
+
```

Comment:
This is something new for us. So fare, we are always used version pattern x.y.z? Where is this new requirement presented?